index.html

```diff
@@ -161,11 +161,31 @@ <h2>Step 3: Run Examples</h2>
                 <p>
                     <h5>00 - Get Authenticated (Set Account ID and BaseUrl)</h5>
                     <p>
-                        The example folder <span class="badgeLabel">00 Authentication</span> contains a single method for finding and storing your <code>accountId</code>, obfuscating your password, and setting your <a href="https://docs.docusign.com/esign/guide/usage/quickstart.html#authenticate-and-get-your-base-url" target="_blank">BaseUrl</a> for subsequent API calls. 
```

Comment:
could have sworn I just sent a commit on my branch that changes this sentence to say Authentication folder as there is no `00` folder.